src/middleware/named.ts

```diff
@@ -1,12 +1,12 @@
-import { ErrorCode, WestEggError } from "../errors";
+import { ErrorCode, WestEggError } from "@gatsby-tv/types";
 
 const DISPLAY_NAME_MIN_LENGTH = 3;
```

Comment:
Display names can be whatever the hell people want for the most part. The maximum length should be 50 characters and must not be an empty string.

Reply:
So minimum of 1 character?

Reply:
fixed